src/sim/trace-extract/google_trace_simulator.cc

```diff
@@ -171,10 +311,46 @@ void GoogleTraceSimulator::Run() {
       LOG(ERROR) << "Could not open bin output file.";
     }
   } else {
-    ReplayTrace();
+    ofstream *stats_file = NULL;
+    if (!FLAGS_stats_file.empty()) {
+      stats_file = new ofstream(FLAGS_stats_file);
+      if (stats_file->fail()) {
+        LOG(FATAL) << "Could not open for writing stats file "
+                   << FLAGS_stats_file;
+      }
+    }
+
+    graph_output_ = NULL;
```

Comment:
I guess we have to make sure we use the same way of writing to files. One uses ofstream and another one uses fopen.